mayastor/src/bdev/dev.rs

```diff
@@ -51,20 +55,14 @@ impl Uri {
         })?;
 
         match url.scheme() {
-            // really should not be used other than for testing
-            "aio" => Ok(Box::new(aio::Aio::try_from(&url)?)),
+            // backend NVMF target - fairly unstable (as of Linux 5.2)
```

Comment:
what does backend thing mean?

Reply:
I preserved the original comment from our current code:
```
// backend NVMF target - fairly unstable (as of Linux 5.2)
"nvmf" => Ok(Box::new(nvmf::Nvmf::try_from(&url)?)),
```

Reply:
backend refers to where we connect to.

Reply:
but does the comment of "fairly unstable (as of 5.2)" refer to the backend or the initiator?

Reply:
We can just remove it, it makes no sense to keep it anymore :+1: